hibernate-core/src/main/java/org/hibernate/dialect/DB2Dialect.java

```diff
@@ -186,6 +186,8 @@ public void initializeFunctionRegistry(QueryEngine queryEngine) {
 		CommonFunctionFactory.median( queryEngine );
 		CommonFunctionFactory.stddev( queryEngine );
 		CommonFunctionFactory.stddevPopSamp( queryEngine );
+		CommonFunctionFactory.regrLinearRegressionAggregates( queryEngine );
+		queryEngine.getSqmFunctionRegistry().registerAlternateKey( "regr_icpt", "regr_intercept" );
```

Comment:
I guess I would leave off `regr_icpt`, since it's not the standard syntax. As long as people can use `regr_intercept()` they're good, it seems to me.

Reply:
Ok, removed in next commit